am-lib/src/main/java/uk/gov/hmcts/reform/amlib/AccessManagementService.java

```diff
@@ -17,4 +18,18 @@ public void createResourceAccess(String resourceId, String accessorId) {
         jdbi.useExtension(AccessManagementRepository.class,
             dao -> dao.createAccessManagementRecord(resourceId, accessorId));
     }
+
+    /**
+     * Returns `resourceJSON` when row with userId (accessorId) and resourceId is in Db, otherwise null
```

Comment:
The caller does not care where it's stored (db).
The message should be more meaningful for someone using this library what is this method for.